romancal/datamodels/ramp.py

```diff
@@ -0,0 +1,41 @@
+from .core import RampDataModel
+#from ..lib.basic_utils import deprecate_class
+
+class RampModel(RomanDataModel):
+    """
+    A data model for 4D ramps.
```

Comment:
Shouldn't this be for 3D ramps, as there will only be 1 integration ?

Reply:
Yup.. and I need to fix it everywhere.

Reply:
Fixed.